src/ActionHandler/WaardepapierenDynamicHandler.php

```diff
@@ -2,16 +2,29 @@
 
 namespace CommonGateway\WaardepapierenBundle\ActionHandler;
 
-use CommonGateway\WaardepapierenBundle\Service\WaardepapierenService;
+use CommonGateway\CoreBundle\ActionHandler\ActionHandlerInterface;
+use CommonGateway\WaardepapierenBundle\Service\WaardepapierService;
 
+/**
+ * WaardepapierenDynamicHandler
+ *
+ * @author   Barry Brands barry@conduction.nl
```

Comment:
Just in general, not specific for this file: don't forget to add your own name & email in the author tags if you worked on it, now or later 😉